configs/components/openssl-1.1.1.rb

```diff
@@ -78,6 +78,9 @@
       target = 'linux-ppc64le'
     elsif platform.architecture =~ /64$/
       target = 'linux-x86_64'
+    # Not a cross-compile for debian-10-arm
+    elsif platform.name =~ /debian-10-arm/
```

Comment:
I know this has been done in a similar way for Debian 8, but it'd be more future-proof to change this `if` to something like `platform.architecture =~ /armhf/`.